lib/ansible/cli/galaxy.py

```diff
@@ -402,8 +402,13 @@ def execute_install(self):
             # query the galaxy API for the role data
 
             if role.install_info is not None and not force:
-                display.display('- %s is already installed, skipping.' % role.name)
-                continue
+                if role.install_info['version'] != role.version:
```

Comment:
I'll put in a LooseVersion check, but it'll have to work as long as new LooseVersion is not less (as we can't compare different branch names etc.)